src/components/ListAndFilterDiscussionForums.js

```diff
@@ -0,0 +1,176 @@
+                  </t>
+                )}
+              >
+                <h1
+                  style={{
+                    fontSize: '1.2em',
+                    width: '100%',
+                    fontWeight: 'bolder',
+                  }}
+                >
+                  <FilterTwoTone style={{ fontSize: '1em' }} />
+                  Filters
+                </h1>
+              </Tooltip>
+            </Menu.Item>
+            <Select
+              onChange={(e) => updateFilters({ type: e })}
+              placeholder="Topics"
+              style={{ width: '95%', padding: '5px 10px' }}
+              mode="multiple"
+              showArrow={true}
+              allowClear={true}
+            >
+              {resourceTypes.map((res) => (
+                <Select.Option value={res}>
+                  {res}
+                </Select.Option>
+              ))}
+            </Select>
+            <Select
+              showSearch
+              onChange={(e) => updateFilters({ path: e })}
+              placeholder="Roles"
+              style={{ width: '95%', padding: '5px 10px' }}
+              mode="multiple"
+              showArrow={true}
+              allowClear={true}
+            >
+              {resourcePath.map((res) => (
+                <Select.Option value={res}>
+                  {res.replace('Path', '')}
+                </Select.Option>
+              ))}
+            </Select>
+            <Menu.Item disbled selectable={false}>
+              <Button href="/discussion">
+                Reset Filters <RedoOutlined style={{ marginRight: '0' }} />
+              </Button>
+            </Menu.Item>
+          </Menu>
+        </Sider>
+      </Affix>
+      <Affix offsetTop={0}>
+      <Select
+              onChange={(e) => updateFilters({ sortBy: e })}
+              placeholder="Sort By"
+              style={{ width: '100%', padding: '5px 10px' }}
+              showArrow={true}
+              allowClear={true}
+            >
+              <Select.Option value="byCreationDateAsc">
+                Date Created (Ascending)
+              </Select.Option>
+              <Select.Option value="byCreationDateDesc">
+                Date Created (Descending)
+              </Select.Option>
+              <Select.Option value="byUpdatedDateAsc">
+                Date Updated (Ascending)
+              </Select.Option>
+              <Select.Option value="byUpdatedDateDesc">
+                Date Updated (Descending)
+              </Select.Option>
+              <Select.Option value="byNameAsc">Name (Ascending)</Select.Option>
+              <Select.Option value="byNameDesc">Name (Descending)</Select.Option>
+              <Select.Option value="byUpvotesAsc">Upvotes (Ascending)</Select.Option>
+              <Select.Option value="byUpvotesDesc">Upvotes (Descending)</Select.Option>
+            </Select>
+      </Affix>
+      <Layout style={{ padding: '24px 24px 24px' }}>

```

Comment:
We should probably wrap the `<Select />` and `<Layout />` in an Ant Design [Grid](https://ant.design/components/grid/) so they're arranged vertically, not next to each other. Its probably as simple as wrapping the Select and the Layout in their own full size rows, something like this:
```
    <Row>
      <Col span={24}><Select  ... /></Col>
    </Row>
    <Row>
      <Col span={24}><Layout  ... /></Col>
    </Row>
```